docs/usage.rst

```diff
@@ -113,6 +113,44 @@ To enable ``hoverxref`` on a domain, you need to use the config :confval:`hoverx
 indicating which are the domains you desire.
 
 
+Tooltip on glossary terms
+-------------------------
+
+You can add tooltips to glossary terms:
+
+.. code-block:: rst
+
+   See the :term:`sphinx:environment` definition in the glossary.
+
+That will render to:
+
+See the :term:`sphinx:environment` definition in the glossary.
+
+To enable ``hoverxref`` on glossary terms, you need to add ``'term'`` to :confval:`hoverxref_roles`.
```

Comment:
Is this enabled by default? I feel like it should be..

Reply:
I prefer to not enable tooltips by default. The extension itself has a `:hoverxref:` role where you can manually specify which references you want to show tooltips on.

Besides, it has a simplified way to enable tooltips on all references by using `hoverxref_auto_ref = True` in your config file or selectively by role using `hoverxref_roles`.